### PR TITLE
Enum fixes found while making #534

### PR DESCRIPTION
**Pull request by NogginBops**
These are some enum fixes that I found while making #534.

### CODE REVIEW

**Comment by SunSerega** [comment]
Same as I said in #534 [here](https://github.com/KhronosGroup/OpenGL-Registry/pull/534#issuecomment-1261516665), I don't think `glSpriteParameterfvSGIX` should get this group, at least for now.
The values of that group are acceptable, but it doesn't make sense to pass enum values as a float when there is an overload for an integer.

**Comment by NogginBops** [comment]
I have removed the group from the parameter now. I can agree that it doesn't make much sense to put the group on that parameter.

**Comment by SunSerega** [comment]
Well, I meant the same for `glSpriteParameterfSGIX` (without 'v'), but I also don't have as strong of an opinion on it because there the workaround is trivial.
And either way this is temporary and a proper system for describing influence of `pname` parameter is needed.

**Comment by NogginBops** [comment]
I missed `glSpriteParameterfSGIX`, fixed!

File: xml/gl.xml [hunk 1]
```diff
@@ -2920,16 +2920,16 @@ typedef unsigned int GLhandleARB;
...
-        <enum value="0x8571" name="GL_COMBINE_RGB" group="TextureEnvParameter"/>
+        <enum value="0x8571" name="GL_COMBINE_RGB" group="TextureEnvParameter,TextureEnvParameter"/>
         <enum value="0x8571" name="GL_COMBINE_RGB_ARB" group="TextureEnvParameter"/>
         <enum value="0x8571" name="GL_COMBINE_RGB_EXT" group="TextureEnvParameter"/>
-        <enum value="0x8572" name="GL_COMBINE_ALPHA" group="TextureEnvParameter"/>
+        <enum value="0x8572" name="GL_COMBINE_ALPHA" group="TextureEnvParameter,TextureEnvParameter"/>
         <enum value="0x8572" name="GL_COMBINE_ALPHA_ARB" group="TextureEnvParameter"/>
         <enum value="0x8572" name="GL_COMBINE_ALPHA_EXT" group="TextureEnvParameter"/>
-        <enum value="0x8573" name="GL_RGB_SCALE" group="TextureEnvParameter"/>
+        <enum value="0x8573" name="GL_RGB_SCALE" group="TextureEnvParameter,TextureEnvParameter"
```
[the remark this answers is not in the document]
Comment (SunSerega) [reply xml/gl.xml:2932]:
```suggestion
        <enum value="0x8573" name="GL_RGB_SCALE" group="TextureEnvParameter,TextureEnvParameter"/>
```

File: xml/gl.xml [hunk 2]
```diff
@@ -2952,56 +2952,56 @@ typedef unsigned int GLhandleARB;
...
         <enum value="0x8599" name="GL_OPERAND1_ALPHA_EXT" group="TextureEnvParameter"/>
-        <enum value="0x859A" name="GL_OPERAND2_ALPHA" group="TextureEnvParameter"/>
+        <enum value="0x859A" name="GL_OPERAND2_ALPHA" group="TextureEnvParameter,TextureEnvParameter"/>
         <enum value="0x859A" name="GL_OPERAND2_ALPHA_ARB" group="TextureEnvParameter"/>
         <enum value="0x859A" name="GL_OPERAND2_ALPHA_EXT" group="TextureEnvParameter"/>
         <enum value="0x859B" name="GL_OPERAND3_ALPHA_NV" group="TextureEnvParameter"/>
             <unused start="0x859C" end="0x859F" comment="Additional combiner enums only"/>
     </enums>
-
+§
```
Comment (SunSerega) [inline xml/gl.xml:3004]: Also this..?

Comment (NogginBops) [reply xml/gl.xml:3004]: Must have fat fingered that...

**Review by Perksey — APPROVED** [review]
I'm happy with this

**Review by SunSerega — APPROVED** [review]

**Comment by SunSerega** [comment]
Since it's connected to #534 - it's also ready to be un-drafted.

**Comment by pdaniell-nv** [comment]
@oddhack this is approved to merge.

**Pull request merged**